app/components/community-bar/local-navigation-dropdown.js

```diff
@@ -78,6 +78,11 @@ export default Component.extend({
 		this.onLinkClicked(event);
 	},
 
+	onSubLevelTitleClicked(event) {
```

Comment:
hmm this looks exactly the same as onThirdLevelSelected do we need another method?

Reply:
maybe rename the method so it could apply to more places?